lib/lxdclient.js

```diff
@@ -314,14 +314,19 @@ function getImageBestFile(img, version) {
         // items but that is not guaranteed to be the case.
         // Discussing with the LXC folks, multiple files of the same ftype would
         // likely result in indeterministic behavior.
+        var itemFromFtype = {};
+
+        Object.keys(items).forEach(function(name) {
```

Comment:
Build is complaining around here...

Reply:
Yeah, just fixed that. New one on the way.